frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -278,12 +130,16 @@ const PropertyListView: React.FC = () => {
    * @param {boolean} getAllFields Enable this field to generate report with additional fields. For SRES only.
    */
   const fetch = (accept: 'csv' | 'excel', getAllFields?: boolean) => {
-    const query = getServerQuery({ pageIndex, pageSize, filter, organizationIds });
+    // Call API with appropriate search parameters
+    const query = toFilteredApiPaginateParams<IPropertyFilter>(
+      pageIndex,
+      pageSize,
+      sort && !isEmpty(sort) ? generateMultiSortCriteria(sort) : undefined,
+      filter,
+    );
     return dispatch(
       download({
-        url: getAllFields
-          ? getAllFieldsPropertyReportUrl({ ...query, all: true, propertyType: undefined })
-          : getPropertyReportUrl({ ...query, all: true, propertyType: undefined }),
+        url: getAllFields ? getAllFieldsPropertyReportUrl(query) : getPropertyReportUrl(query),
```

Comment:
Not sure if this getAllFields flag makes sense any more, did they mention this in the requirements?

Reply:
Nobody has mentioned this but was not sure what to take out without breaking existing functionality. If you know more please provide context here and will be happy to remove/change. Not sure 100% if export endpoints work to be hones

Reply:
There is this comment:
@param {boolean} getAllFields Enable this field to generate report with additional fields. For SRES only.

So it sounds like some fields were locked to the SRES role. I don't think we have any hidden fields that are role locked currently, so probably can remove this.

Reply:
based on testing they seem to work ok. Not sure if the list of columns is correct, but if you have no requirements on that I'd leave it as is.